beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/README.md

```diff
@@ -1,2 +1,106 @@
-# gdc-wgs
-WDL implementation of the GDC Somatic WGS alignment pipeline, documentation to come.
+# Genomic Data Commons (GDC) Whole Genome Somatic Single Sample Overview
+
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [GDCWholeGenomeSomaticSingleSample_v1.0.0](https://github.com/broadinstitute/warp/releases) | January, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+
+## Introduction to the GDC Whole Genome Somatic Single Sample Pipeline
+The GDC Whole Genome Somatic Single Sample (abbreviated GDC here) pipeline is the alignment and preprocessing workflow for genomic data designed for the National Cancer Institute's [Genomic Data Commons](https://gdc.cancer.gov/about-gdc). 
+
+A high-level overview of the pipeline in addition to tool parameters are available on the [GDC Documentation site](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/). 
+
+Overall, the pipeline converts reads (CRAM or BAM) to FASTQ and (re)aligns them to the latest human reference genome. Each read group is aligned separately. Read group alignments that belong to a single sample are then merged and duplicate reads are flagged for downstream variant calling. 
+
+
+## Set-up
+
+### Workflow Installation and Requirements
+
+The workflow is written in the Workflow Description Language WDL and can be downloaded by cloning the [warp repository](https://github.com/broadinstitute/warp/tree/master) in GitHub. The workflow can be deployed using [Cromwell](https://github.com/broadinstitute/cromwell), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. 
+
+For the latest workflow version and release notes, please see the [changelog](https://github.com/broadinstitute/warp/blob/master/beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md).
+
+### Software Version Requirements
+
+* GATK 4.0.7
+* Picard 2.18.11 (Custom Docker is used to run software on Cromwell 52)
+* Samtools 1.11
+* Python 3.0
+* Cromwell version support
+    * Successfully tested on v52
+    * Does not work on versions < v23 due to output syntax
+* Papi version support
+	* Successfully tested on Papi v2
+
+### Input Descriptions
+
+The table below describes each of the GDC pipeline inputs. The workflow requires a **single** aligned CRAM or BAM file, or a **single** unmapped BAM (uBAM) as input.
```

Comment:
Do we normally include information about which inputs are optional versus required? It might be helpful (?)

Reply:
Updated the inputs to reflect optional inputs